.azurepipelines/BaseTools-Build-For-Publication.yml

```diff
@@ -0,0 +1,146 @@
+name: BaseTools_Bin_$(release_version)_$(Rev:r)
+
+variables:
+  linux_build_output_path: '$(Build.Repository.LocalPath)/BaseTools/Source/C/bin'
+  linux_artifact_name: 'LinuxOutput'
+  windows_build_output_path: '$(Build.Repository.LocalPath)\BaseTools\Bin\Win32'
+  windows_artifact_name: 'WindowsOutput'
+  temp_publication_directory: 'PackageStaging'
+  package_artifact_name: 'NugetReleasePackage'
+
+jobs:
+  - job: linux_build
+    displayName: Linux Build
+    pool:
+      vmImage: ubuntu-latest
+
+    workspace:
+      clean: all
+
+    steps:
+    - checkout: self
+      clean: true
+  
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.7.x'
+        architecture: 'x64'
+      displayName: Install correct Python
+
+    - script: pip install -r requirements.txt --upgrade
+      displayName: 'Install/Upgrade pip modules'
+
+    # Set default
+    - bash: |
+        echo "##vso[task.setvariable variable=pkg_count]${{ 1 }}"
```

Comment:
unless your goal is to keep aligned with edk2 this seems unnecessary

Reply:
The template fails to run if this isn't set...
I could roll this into a new template and knock out both birds with one stone...